.circleci/config.yml

```diff
@@ -89,7 +97,7 @@ jobs:
       - run:
           name: Publish npm package
           command: |
-            if [ -n "${npm_TOKEN+x}" ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
+            if [ -n "${npm_TOKEN}" ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
This change looks strange, is it correct?

Reply:
yes, as `-n` `test` function already correctly handles the values `+x` filter has targeted on